sql/sys_vars.cc

```diff
@@ -7391,3 +7392,18 @@ static Sys_var_enum Sys_block_encryption_mode(
   "AES_ENCRYPT() and AES_DECRYPT() functions",
   SESSION_VAR(block_encryption_mode), CMD_LINE(REQUIRED_ARG),
   block_encryption_mode_values, DEFAULT(0));
+
+extern ulonglong opt_binlog_commit_by_rotate_threshold;
+static Sys_var_ulonglong Sys_binlog_large_commit_threshold(
+  "binlog_large_commit_threshold",
```

Comment:
I believe we also have to make it clear in [--tmpdir docs lines](https://github.com/MariaDB/server/blob/39d134b19907af1368e037e97939aa2f3e6169d7/sql/sys_vars.cc#L3259) that the user thread's binlog IO_Cache temporary files are relocated into a folder in the same directory with the binlog files.

Reply:
```
       "tmpdir", "Path for temporary files.
+      Files that are created in background for binlogging by user threads are placed in a separate location (see `binlog_large_commit_threshold` option).
```

Reply:
added